larq_compute_engine/tflite/kernels/bmaxpool.cc

```diff
@@ -27,8 +27,8 @@ void* Init(TfLiteContext* context, const char* buffer, size_t length) {
 
   poolparams->filter_height = m["filter_height"].AsInt32();
   poolparams->filter_width = m["filter_width"].AsInt32();
-  poolparams->stride_height = m["stride_height"].AsInt32();
-  poolparams->stride_width = m["stride_width"].AsInt32();
+  poolparams->stride_height = m["stride_h"].AsInt32();
```

Comment:
a nasty bug. great that you cached it on Pixel runs. can we make the naming scheme consistent on the converter side? @AdamHillier @lgeiger

Reply:
Yeah I think we should stick to the TFLite naming scheme as close as possible. Currently this is not the case for the BConvOp which still follows the naming of the TF operations.

Reply:
I just chatted to @Tombana about this, I'm happy to change it in the converter. For reference, `stride_h` is what is used in TFL but as Tom pointed out it does seem inconsistent with e.g. `filter_height`.